include/pru/am572x.h

```diff
@@ -0,0 +1,769 @@
+/* SPDX-License-Identifier: BSD-2-Clause-FreeBSD
+ *
+ * pru/am335x.h -- PRU I/O registers definitions for PRU inside AM335x SoCs
+ *
+ * Copyright (c) 2020 Vedant Paranjape
```

Comment:
You have reused am335x.h [1] and that is perfectly OK in the free software world. But please retain previous copyright, or mention  the base you have used if your changes are substantial:

```
 * Based on include/pru/am335x.h
```

[1] Command that showcases copied files:
```
$ git show --find-copies-harder 59fcd8e9db794f4cecc3e27bac82c279bd54873a
```

Reply:
Is there any way I can extend your license? like Dinuxbg is the original owner, then Vedant made the changes to it ?

Reply:
Updated in 7aaee30f69a71f0ade202294305049365e12f2fe. Let me know if it will work !

Reply:
This should be enough:

```
 * pru/am572x.h -- PRU I/O registers definitions for PRU inside AM527xx SoCs
 * Based on pru/am335x.h
 *
 * Copyright (c) 2020 Vedant Paranjape
```